com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/ReflectionMenuItem.cs

```diff
@@ -27,7 +27,7 @@ static void CreateMirrorGameObject(MenuCommand menuCommand)
             }
         }
 
-        [MenuItem("GameObject/Light/Planar Reflection Probe", priority = CoreUtils.gameObjectMenuPriority)]
+        [MenuItem("GameObject/Light/Planar Reflection Probe", priority = 22)]
```

Comment:
why do we change to a fixed number and not _CoreUtils.Sections.section2 + CoreUtils.Priorities.gameObjectMenuPriority_ for example?

Reply:
This is the only item that goes under `Light`, I thought it would be more clear to have just the hardcoded number here instead of adding a new field on the `CoreUtils.Priorities`.